CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ Releases 4.5.x and 4.6.x are based on mainline support submitted in
 [this gerrit ref](https://review.coreboot.org/#/c/14138/).
 
 ## [Unreleased]
+## [v4.0.30] - 2020-02-28
```

Comment:
@piotr-kleins please change the date to be today. I will push a tag by the end of the day so that the regression tests will finish over the night

Reply:
done